test/integration/SymfonyMimeMimeTypeGuesserIntegrationTest.php

```diff
@@ -4,17 +4,16 @@
 
 namespace Boesing\Psr\Http\Message\Multipart;
 
+use Boesing\Psr\Http\Message\Multipart\SymfonyMimeMimeTypeGuesser;
```

Comment:
shouldn't be necessary as the namespace is already `Boesing\Psr\Http\Message\Multipart` but I guess that has been added by codesniffer? 🤷🏼‍♂️